portal-ui/src/screens/Console/Buckets/ViewBucket/ViewBucket.tsx

```diff
@@ -379,6 +389,18 @@ const ViewBucket = ({
     setLoadingEncryption(true);
   };
 
+  const closeAddModalAndRefresh = (refresh: boolean) => {
+    setPolicyScreenOpen(false);
+
+    if (refresh) {
+      fetchRecords();
+    }
+  };
+
+  const fetchRecords = () => {
```

Comment:
Please change this to fetchPolicies
```suggestion
  const fetchPolicies = () => {
```